display_list/display_list_canvas_dispatcher.cc

```diff
@@ -72,6 +73,27 @@ void DisplayListCanvasDispatcher::saveLayer(const SkRect* bounds,
   }
 }
 
+void DisplayListCanvasDispatcher::saveLayerCF(
```

Comment:
Why does this method exist? It doesn't do anything that saveLayer(... paint.colorFilter) can already do. Please delete it.